extensions/sdk/src/Rally.ts

```diff
@@ -267,16 +271,10 @@ export class Rally {
 
           const data = userStudiesDoc.data();
 
-          if (data.enrolled && this._state !== RunStates.Running) {
+          if (data.enrolled) {
```

Comment:
Why did we remove running check here?

Reply:
Hm, yeah I agree, this was preventing `resume()` from being called if the extension was already running. We might want to consider moving to a proper state machine that checks this vs. doing it ad-hoc like this.

Reply:
`resume()` already has this same check in it's implementation, so I figured it was redundant.